### ISSUE TITLE
Dock on inner/outer edges for dual screen setup.

### ISSUE THREAD

**Issue by bartoszek** [micheleg/dash-to-dock#705]
Hi,
I would like to have an option to set up docks on my dual vie machine to be located on opposite edges of the screen.
My pitch for this feature.
![dash-to-dock-pitch](https://user-images.githubusercontent.com/1213348/37519312-e87739b8-2918-11e8-8ab5-c6c513c72910.jpg)

**Comment by micheleg** [comment]
I totally understand the point. I think it could be indeed integrated as an "outer edge" option (not sure about the inner case).

Have a look at this branch, I quickly tried to hack together this feature (with no settings) yet, and the heuristic to identify the monitor positions might need some re-thoughts.

https://github.com/micheleg/dash-to-dock/tree/experimental_outer_dock

Could you test it and see if there's any bug or improvements?

**Comment by szczepaniak-bartek** [comment]
For now, seems to works fine, on my system. Thanks for quick response.
Notice some misbehaviour:
* top/bottom always shows on top.
* left/right always shows on the outer edges.
I presume they are result of quick, bodged implementation of the feature ;)

**Comment by micheleg** [comment]
Yes, I hard-coded the behavior as a proof of concept. I think from my quick test I also noticed some issue with the show app buttons menu on the right side dock.

**Comment by kowalcj0** [comment]
:+1: from me.
I'd love to see that feature as well.

**Comment by williansdejulio** [comment]
That feature would really help me too 👍

**Comment by kowalcj0** [comment]
Hi, 
This is the killer feature I'm looking for! 
Any chances of reviving it?
Thanks
